src/Components/ChatRoom/index.js

```diff
@@ -1,60 +1,29 @@
-import React from "react";
-import MyMessag from "./../MyMsg/index.js";
-import OtherMessage from "./../OtherMsg/index.js";
-import MyBigMsg from "./../MyBigMsg/index.js";
-import { Container, ChatRoomHead, Title, UlLIst, UlLIstLi,
-         Head, HeadPFirst, HeadPSecond, ChatMessageList,
-         Footer, Add, Send, InputContainer, Input, Date, DateP } from "./style.js";
+import React            from "react";
+import MyBigMsg         from "./../MyBigMsg/index.js";
+import ChatRoomHead     from "./../ChatRoomHead/index.js";
+import ChatRoomSupitle  from "./../ChatRoomSupitle/index.js";
+import ChatRoomDate     from "./../ChatRoomDate/index.js";
+import ChatRoomFooter   from "./../ChatRoomFooter/index.js";
+import MsgContent       from "./../MsgContent/index.js";
+import { Container,
+      ChatMessageList } from "./style.js";
 
 function ChatRoom() {
   return (
     <Container>
-      <ChatRoomHead>
-        <Title>Mom's Discharge Plan</Title>
-        <UlLIst>
-          <UlLIstLi>
-            <a href="null">
-              <img src="images/ico_group.png" alt="" />
-            </a>
-          </UlLIstLi>
-          <UlLIstLi>
-            <a href="null">
-              <img src="images/ico_filter.png" alt="" />
-            </a>
-          </UlLIstLi>
-          <UlLIstLi>
-            <a href="null">
-              <img src="images/ico_search.png" alt="" />
-            </a>
-          </UlLIstLi>
-        </UlLIst>
-      </ChatRoomHead>
+      <ChatRoomHead />
 
-      <Head>
-        <HeadPFirst>Starter Care Plan created + daily actions added</HeadPFirst>
-        <HeadPSecond>12:10p</HeadPSecond>
-      </Head>
+      <ChatRoomSupitle />
 
       <ChatMessageList>
-        <MyMessag />
-        <OtherMessage />
-        <Date>
-          <DateP>Apr 1</DateP>
-        </Date>
+        <MsgContent />
+        
+        <ChatRoomDate />
+
         <MyBigMsg />
```

Comment:
Shouldn't MyBigMsg appear first in the order as it's before the two other messages?